lib/spandx/cli.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
+require 'nanospinner'
```

Comment:
@mokhan why we prefer this over tty-screen ?

Reply:
I met the author. I like that it has no dependencies and that it is simple. The output from tty-progressbar is very nice but when scanning a large directory it becomes difficult to follow the output. Providing an accurate % of progress is difficult and requires knowing the total size of operations which makes streaming output concurrently difficult. Instead of calculating a % nanospinner displays a spinner to indicate that work is being done with minimal output. The next step is to start breaking up the scan steps so that work can be done concurrently. I think nanospinner works nicely with this approach. Also, did I mention that I met the author. I like their work. It's nano.